Source/WebCore/platform/graphics/cg/ImageBufferIOSurfaceBackend.cpp

```diff
@@ -130,10 +129,10 @@ void ImageBufferIOSurfaceBackend::flushContext()
 
 bool ImageBufferIOSurfaceBackend::flushContextDraws()
 {
-    if (!m_context)
-        return false;
-    if (!m_context->consumeHasDrawn())
+    bool contextNeedsFlush = m_context && m_context->consumeHasDrawn();
```

Comment:
This seems ok, while unfortunate...

Conceptually these are :
-  "GraphicsContextCG has drawn to the surface" -- `consumeHasDrawn()`
 - "CGContext has drawn to the surface" -- `std::exchange(m_needsFirstFlush, false)`